lms/models/organization.py

```diff
@@ -0,0 +1,42 @@
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+
+from lms.db import BASE
+from lms.models._mixins import CreatedUpdatedMixin
+from lms.models.application_settings import ApplicationSettings
+
+
+class Organization(CreatedUpdatedMixin, BASE):
+    """Model for Organizations comprised of application instances."""
+
+    __tablename__ = "organization"
+
+    id = sa.Column(sa.Integer(), autoincrement=True, primary_key=True)
```

Comment:
We talked about having a public id here.

The most common solution it's probably having a `uuid = ` with a generated UUID here.

The `uuid` name is probably a bad idea to avoid confusion with GUIDs. 

The value could still be a UUID, maybe b64'ed so it looks different?

```b"US-" + base64.urlsafe_b64encode(uuid.uuid4().bytes).replace(b"=", b"")```

Reply:
I went with something very similar after _many_ attempts at seeing if I could do better. Just so it doesn't feel like a total waste:

 * There's Base86 in the same module, but the output is too wild and would probably cause problems in URLs
 * I tried rolling my own Base-Many, but there aren't that many more printable characters you'd want to use, so it doesn't end up any shorter
 * The other types of UUID aren't of any use
 * There are various "short-code" generators, but they probably aren't going to justify their existence with regards to a new dep
 
We might want to consider "slugifying" vanity ids if/when we allow people to change them manually. We already use "python-slugify" in other places. We can't "slugify" these UUIDs as it will modify them allowing for clashes.